cmd/conformance/gcp/main.go

```diff
@@ -37,7 +37,6 @@ var (
 	listen            = flag.String("listen", ":2024", "Address:port to listen on")
 	spanner           = flag.String("spanner", "", "Spanner resource URI ('projects/.../...')")
 	signer            = flag.String("signer", "", "Note signer to use to sign checkpoints")
-	verifier          = flag.String("verifier", "", "Note verifier corresponding to --signer")
```

Comment:
You'll need to update (and possibly deploy) `deployment/modules/gcp/conformance/main.tf`

Reply:
Done - I think we should be good without deploying, right? Cloudbuild should just pick everything up?

Reply:
@AlCutter for the right answer - I can't recall which way this is deployed right now.

Reply:
So long as this config always matches the flags in the binary I think it should be good as the conformance test is currently [re-]created from scratch each time.